src/format/opamFile.ml

```diff
@@ -1297,6 +1299,7 @@ module ConfigSyntax = struct
     { t with installed_switches }
   let with_switch_opt switch t = { t with switch }
   let with_switch switch t = { t with switch = Some switch }
+  let with_previous_switch previous_switch t = { t with previous_switch }
```

Comment:
According the convention, `with_x` takes `'x` value, while `with_x_opt` takes `'x option` value (like `with_switch`).